caselaw/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
```

Comment:
We should not be committing settings.py. At least, we should make clear this file is only settings for local development or an example settings file.